panels/zcalibrate.py

```diff
@@ -90,7 +92,19 @@ def activate(self):
             y_position = self._config.get_config()['z_calibrate_position'].getint("calibrate_y_position", 0)
             if x_position > 0 and y_position > 0:
                 self._screen._ws.klippy.gcode_script('G0 X%d Y%d F3000' % (x_position, y_position))
-        self._screen._ws.klippy.gcode_script(KlippyGcodes.PROBE_CALIBRATE)
+
+        if (self._printer.config_section_exists("probe") or self._printer.config_section_exists("bltouch")):
+            self._screen._ws.klippy.gcode_script(KlippyGcodes.PROBE_CALIBRATE)
+        else:
+            self._screen._ws.klippy.gcode_script(KlippyGcodes.Z_ENDSTOP_CALIBRATE)
+
+        self.start.get_style_context().remove_class('color3')
```

Comment:
Can you change these to be in the self.labels dictionary? It may be inappropriately named at this point, but it's where widgets are stored. I can update the name in the future to self.widgets